openapi_server/apis/users_api.py

```diff
@@ -138,7 +137,18 @@ async def get_favorite_foods(
     username: str = Path(None, description="Name of user"),
 ) -> ListFavoriteFoods:
     """"""
-    ...
+    try:
+        user: DbUser = DbUser.get(username)
+    except DbUser.DoesNotExist:
+        raise HTTPException(status_code=404)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+    favorite_foods: List[FavoriteFood] = []
+    for item in user.favorite_foods:
+        food: FavoriteFood = FavoriteFood(id=item.id, name=item.name)
+        favorite_foods.append(food)
+    return ListFavoriteFoods(favoriteFoods=favorite_foods)
```

Comment:
we can simplify the code using lambda and map function

`favorite_foods: List[FavoriteFood] = list(map(lambda food: FavoriteFood(id=food.id, name=food.name), user.favorite_foods))`

Reply:
Oh I see. This seems to be the Python equivalent of Javascript's arrow and map function.